Applications/DataExplorer/mainwindow.cpp

```diff
@@ -624,18 +624,22 @@ void MainWindow::loadFile(ImportFileType::type t, const QString &fileName)
     }
     else if (t == ImportFileType::TETGEN)
     {
-        if (fi.suffix().toLower().compare("poly") == 0 || fi.suffix().toLower().compare("smesh") == 0)
+        if (fi.suffix().toLower().compare("smesh") == 0)
```

Comment:
i excluded it because They the Tetgen interface is not supporting poly format yet.